packages/portfolio-contract/src/portfolio.exo.ts

```diff
@@ -499,8 +507,8 @@ export const preparePortfolioKit = (
       },
       rebalanceHandler: {
         async handle(seat: ZCFSeat, offerArgs: unknown) {
-          const { reader, manager } = this.facets;
-          const keeper = { ...reader, ...manager };
+          const { reader, manager, reporter } = this.facets;
+          const keeper = { reader, manager, reporter };
           mustMatch(offerArgs, OfferArgsShapeFor.rebalance);
           return rebalance(seat, offerArgs, keeper);
```

Comment:
weird! how did that ever work?!

But let's not call the record of facets `keeper`:

```suggestion
          const { reader, manager, reporter } = this.facets;
          mustMatch(offerArgs, OfferArgsShapeFor.rebalance);
          return rebalance(seat, offerArgs, { reader, manager, reporter });
```